standards/storing-source-code.md

```diff
@@ -27,9 +27,31 @@ email address to your account](https://help.github.com/articles/adding-an-email-
 and maybe [use that for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
 Ask your delivery manager to request you being added to the Github organisation.
 
-Repositories should be [clearly named]({{ '/standards/naming-things' | relative_url }}),
-and have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})
-and enough documentation that someone new can get started with the
-project.
-
-Private repositories are not a good way to protect secrets, and should only be used where access to the code might reveal draft policy decisions.  Secrets should be managed at the platform level.
+Private repositories are not a good way to protect secrets, and should only be used where access to the code might reveal draft policy decisions. Secrets should be managed at the platform level.
+
+If a repository becomes redundant and the codebase is no longer in use or required then it should be archived for # months before it is deleted altogether. This should be done following agreement with relevant parties.
```

Comment:
What is the feeling as to what the "#" should represent here? 6 months, 12 months?

Reply:
Hi Dan, updated to state that for public repos we should archive for 12 months and for private 6 months (so that we can free up private slots quicker). I've also added that we have 90 days after deletion to recover as well.